app/frontends/jobs.ts

```diff
@@ -47,6 +47,13 @@ function getLinksForDisplay(job: Job, urlRoot: string, statusLinkRel: string): J
   if (job.status === JobStatus.SUCCESSFUL && needsStacLink(dataLinks)) {
     links.unshift(new JobLink(getStacCatalogLink(urlRoot, job.jobID)));
   }
+  if (job.status === JobStatus.PAUSED) {
+    links.unshift(new JobLink({
+      title: 'Resumes the job.',
+      href: `${urlRoot}/jobs/${job.jobID}/resume`,
+      type: 'application/json',
```

Comment:
We try to have a `rel` attribute on most of the links - maybe call this one `rel: resume` so it is is easier to programmatically find the resume link.

https://www.iana.org/assignments/link-relations/link-relations.xhtml has a list of official ones, but I didn't find anything good in there with a quick check.

Reply:
It sounds stupid, but I think `rel: resumer` would be more in line with the way the relationships are named